app/forms/draft_work_form.rb

```diff
@@ -79,6 +82,16 @@ def unique_filenames
     errors.add(:attached_files, 'must all have a unique filename.') unless filenames.size == filenames.uniq.size
   end
 
+  # Ensure there is more than zero files if fetching from Globus
+  def globus_files_provided
+    return if GlobusClient.get_filenames(path: work_version.globus_endpoint, user_id: work.owner.email).any?
+
+    errors.add(
+      :attached_files,
+      "must include at least one file uploaded to Globus at #{work_version.globus_endpoint_fullpath}"
+    )
+  end
```

Comment:
Use existing form machinery to validate the new requirement and present errors to the user where they expect to see them.